full-service/src/json_rpc/mod.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020-2021 MobileCoin Inc.
+
+//! JSON-RPC-2.0 API
+
+mod account;
+mod address;
+mod block;
+mod json_rpc_request;
+mod json_rpc_response;
+mod membership_proof;
+mod transaction_log;
+mod txo;
+mod wallet_api;
+mod wallet_status;
+
+pub use account::Account;
+pub use address::Address;
+pub use block::{Block, BlockContents};
+pub use json_rpc_request::Request;
+pub use json_rpc_response::BalanceResponse;
```

Comment:
Looks like this repo doesn't have a proper rustfmt.toml file, or that rustfmt wasn't ran. We group imports in other places (for example line 27).

Reply:
Good catch. I just added a rustsfmt.toml to the project.